Syntax error and undefined plugin option variable

New projects created from this repository will fail to compile with two errors: 

```
$ sensu-plugin-tool new check foo
? Template URL https://github.com/sensu-community/check-plugin-template
? Project name foo
? Description foo
? Github User foo
? Github Project foo
? Copyright Year 2020
? Copyright Holder foo inc.
Success!
$ cd foo/
$ go build
# github.com/foo/foo
./main.go:26:10: syntax error: unexpected :=, expecting =
$ vim main.go 
$ go build
# github.com/foo/foo
./main.go:52:48: undefined: checkOptions
$ vim main.go 
$ go build
$ ./foo 
Usage:
  foo [flags]
  foo [command]

Available Commands:
  help        Help about any command
  version     Print the version number of this plugin

Flags:
  -e, --example string   An example string configuration option
  -h, --help             help for foo

Use "foo [command] --help" for more information about a command.

Error executing foo: error validating input: --example or CHECK_EXAMPLE environment variable is required
```

--- a/main.go
+++ b/main.go
@@ -23,7 +23,7 @@ var (
 		},
 	}
 
-	options := []*sensu.PluginConfigOption{
+	options = []*sensu.PluginConfigOption{
 		&sensu.PluginConfigOption{
 			Path:      "example",
 			Env:       "CHECK_EXAMPLE",
@@ -49,6 +49,6 @@ func checkArgs(event *types.Event) (int, error) {
 }
 
 func executeCheck(event *types.Event) (int, error) {
-	log.Println("executing check with --example", checkOptions.Example)
+	log.Println("executing check with --example", plugin.Example)
 	return sensu.CheckStateOK, nil
 }

